src/sources/internal_logs.rs

```diff
@@ -64,26 +65,26 @@ async fn run(
         .chain(tokio_stream::wrappers::BroadcastStream::new(
             subscription.receiver,
         ))
+        .filter_map(|log| future::ready(log.ok()))
+        .filter(|log| future::ready(!log.contains("internal")))
```

Comment:
It looks like this filters any logging in `vector` that includes an `internal` field from appearing in `internal_logs`? I'm not sure if there's precedence for this already, but, if not, it may be worth documenting somewhere. That said, I'm not sure where would be best.

Reply:
I'll see if @bruceg has a better option in mind (not to fight with the documentation to drop it after)